### PR TITLE
separate start and end times

### PR DESCRIPTION
**Pull request by jack-ary**

### CODE REVIEW

File: packages/react-frontend/src/Search.js [hunk 1]
```diff
@@ -113,12 +162,32 @@ function Search(props) {
...
+          }
+        });
+      }
+
+    function isEndTimeValid(startTime, endTime) {
+        const startDate = new Date(`2023-01-01 ${startTime}`);
+        const endDate = new Date(`2023-01-01 ${endTime}`);
+        
+        return startDate <= endDate;
     }
```
Comment (Supriya-Mandava) [inline packages/react-frontend/src/Search.js:191]: Consider making the hardcoded date into a constant or parameter for more future flexibility in the function

**Comment by spruhaN** [comment]
looks good to me! maybe we can add a dash in between the two dates to really show that its a range.

**Comment by ivanreet507** [comment]
Looks good to me! Maybe consider selectively displaying the end time options based on the chosen start time to prevent the selection of an end time that precedes the start time.

**Comment by jack-ary** [comment]
> Looks good to me! Maybe consider selectively displaying the end time options based on the chosen start time to prevent the selection of an end time that precedes the start time.

@ivanreet507 Good suggestion. I was struggling to implement this but can do it in a later PR next week.

File: packages/react-frontend/src/Search.js (hunk 1 above)
Comment (jack-ary) [reply packages/react-frontend/src/Search.js]: Added in

**Comment by jack-ary** [comment]
> looks good to me! maybe we can add a dash in between the two dates to really show that its a range.

Added in

**Pull request merged**